windows/src/robocorp/windows/_control_element.py

```diff
@@ -30,6 +30,17 @@
 DEFAULT_SEND_KEYS_INTERVAL = 0.01
 
 
+class _SentinelValidator:
```

Comment:
Doesn't hurt to have a docstring explaining that this is used to be replaced with a default function when `validator` param's value stays unchanged.